joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -1199,10 +1243,6 @@ TEST_F(TrajectoryControllerTest, incorrect_initialization_using_interface_parame
   command_interface_types_ = {"bad_name"};
   set_parameter_and_check_result();
 
-  // command interfaces: effort not yet implemented
-  command_interface_types_ = {"effort"};
```

Comment:
It would be then great to add here checks for illegal configurations, as mentioned in the comment above.

Reply:
The same for velocity-only mode would be nice to have.